simple_switch/l2_switch_grpc/Makefile

```diff
@@ -1,4 +1,4 @@
-P4_SRC = l2_switch.p4
+P4_SRC = l2_switch_grpc.p4
```

Comment:
Renaming the source file breaks the scion/topo script. Please change to invokation of the controller to use l2_switch_grpc.json instead of l2_switch.json to make it work.

Reply:
Changed the scion/topo script to work with the renamed files (uses now the name given for the script's SWITCH variable inside the controller invocation).